Doc/library/enum.rst

```diff
@@ -998,13 +998,18 @@ Finer Points
 Supported ``__dunder__`` names
 """"""""""""""""""""""""""""""
 
-:attr:`__members__` is an :class:`OrderedDict` of ``member_name``:``member``
+:attr:`__members__` is a read-only ordered dictionary ``member_name``:``member``
 items.  It is only available on the class.
 
 :meth:`__new__`, if specified, must create and return the enum members; it is
 also a very good idea to set the member's :attr:`_value_` appropriately.  Once
 all the members are created it is no longer used.
 
+.. versionchanged:: 3.8
+   :attr:`__members__` is changed from :class:`collections.OrderedDict` to
```

Comment:
Oh, I didn't notice about difference of `__members__` and `_member_map_`.
If `_member_map_` is private and not documented, could I remove this `versionchanged` section?
I think documenting it can confuse readers.